### ISSUE TITLE
Add a url but setting the anchor's text

### ISSUE THREAD

**Issue by Nerian** [Paxa/fast_excel#69]
Hi,

We can add add urls like this:
```
worksheet.append_row([
  FastExcel::URL.new("https://github.com/Paxa/fast_excel"),
  FastExcel::URL.new("postgres://localhost")
], url_format)
```

But how can we add a url to the excel file and set the text that will be the anchor to the url? Essentially I want to show a title, and when the user clicks the title it goes to a url.

How can I do this?

**Comment by joshuapinter** [comment]
Yes PLEASE! This would be super useful as often times (for us) the link is a giant and nasty expiring asset link to AWS or something and we want to use the cleaner filename as the anchor text.

**Comment by AlexandreBernard** [comment]
It seems you can use `worksheet.write_url_opt(row, index, "https://github.com", nil, "Github", nil)`

it's undocumented but it works

https://github.com/Paxa/fast_excel/blob/master/lib/fast_excel/binding/worksheet.rb#L265

**Comment by joshuapinter** [comment]
@AlexandreBernard If that works then I wonder if we can write a PR to add a nicer helper method like @Nerian proposed: 

```
FastExcel::URL.new("Super link", "https://github.com/Paxa/fast_excel")
```

Thoughts?

**Comment by AlexandreBernard** [comment]
I just had a look, that's doable I guess

1. change URL class in lib/fast_excel.rb for something like that

```
class URL
    attr_accessor :url, :label
    def initialize(url, label = nil)
      @url = url
      @label = label
    end
  end
```

2. in the same file around L475
https://github.com/Paxa/fast_excel/blob/master/lib/fast_excel.rb#L475

```
...
      elsif value.is_a?(FastExcel::URL)
        if value.label
          write_url(row_number, cell_number, value.url, format)
        else
          write_url_opt(row_number, cell_number, value.url, format, value.label, nil)
        end
        add_text_width(value.url, format, cell_number) if auto_width?
      else
...
```

I don't know what the latest option "tooltip" is but it could be implemented as well

it could be written a nicer way but it should work, I guess (IMHO write_url_opt could be merged into write_url for example)

Feel free, I'm on a day off today 😂
